src/events/kyuubi.py

```diff
@@ -94,3 +107,54 @@ def _on_peer_relation_departed(self, event: ops.RelationDepartedEvent):
         This is necessary for updating status of all units upon scaling up/down.
         """
         self.logger.info("Kyuubi peer relation departed...")
+
+    @compute_status
+    def _on_secret_changed(self, event: SecretChangedEvent) -> None:
```

Comment:
**question**: Am I missing something, or are we not "observing" the event triggering this handler?

Reply:
I would have the same question actually... :D